base_tier_validation/models/tier_validation.py

```diff
@@ -632,6 +634,21 @@ def get_view(self, view_id=None, view_type="form", **options):
                         all_models[model] = res["models"][model]
                 new_node = etree.fromstring(new_arch)
                 node.append(new_node)
+            excepted_fields = self._get_under_validation_exceptions()
+            for node in doc.xpath("//field[@name][not(ancestor::field)]"):
+                if node.attrib.get("name") in excepted_fields:
+                    continue
+                modifiers = json.loads(
+                    node.attrib.get("modifiers", '{"readonly": false}')
+                )
+                if modifiers.get("readonly") is not True:
+                    modifiers["readonly"] = OR(
+                        [
+                            modifiers.get("readonly", []) or [],
+                            [("review_ids", "!=", [])],
+                        ]
+                    )
+                    node.attrib["modifiers"] = json.dumps(modifiers)
```

Comment:
@hbrunn Am I missing something or the fields stay readonly, even if the reviews are validated or rejected ?

Reply:
that's intentional. The goal is: Make it so you can't edit anything after requesting validation.
Where is that a problem?

Reply:
Thank you for your answer. 
On my opinion, the review process is depending of the record state. After 1/ changing the record state and 2/validate or reject the review, the record need to be editable again (not readonly anymore) no ?
For exemple, if the fields are readonly only if a review_ids state is "pending" ?

Reply:
I don't really see why you would want to allow editing for validated records. Doesn't that miss the point of validation? For validation rejected, review_ids is deleted once you restart the validation, so you can edit again. What's your use case?

If this is for some custom code of yours, maybe propose a refactoring of the for loop to call methods for its different aspects, and then you can override/customize this part in that method?

Reply:
For exemple, for a product with 3 states : draft, normal, end of life. 
We validate a product in draft and put it in normal state. Then, we can not change the state as end of life. 
Sorry to insist, I see your point of view and I m agree but the module is based on the state of the record and between 2 states only. 
I will be happy to propose a refactor or a modification depending of your advices.

Reply:
to me that would make the eol field an excepted field?

Otherwise, I see several options:
- don't look at review_ids here, but at validation_state == pending
- make it configurable for programmers with a flag on the class
- make it configurable for programmers by refactoring this into its own function

Reply:
Thank you for your time, I like the last option, I made a PR : https://github.com/OCA/server-ux/pull/888